src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.conteiner {
```

Comment:
There is a typo in the class selector '.conteiner'. It should be '.container' to match the corrected class name in the HTML file.